src/SFA.DAS.AssessorService.Application.Api/Controllers/RegisterQueryController.cs

```diff
@@ -62,6 +62,27 @@ public async Task<IActionResult> GetAssessmentOrganisations()
             return Ok(await _mediator.Send(new GetAssessmentOrganisationsRequest()));
         }
 
+        [HttpGet("assessment-organisations/list", Name = "GetAssessmentOrganisationsList")]
+        [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(List<AssessmentOrganisationSummary>))]
+        [SwaggerResponse((int)HttpStatusCode.BadRequest, Type = typeof(IDictionary<string, string>))]
+        [SwaggerResponse((int)HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
```

Comment:
Do we need these since the method would return an Ok except for extenuating circumstances?

Reply:
It's a pattern used in existing methods so will leave them there

Reply:
If a tester uses the Swagger interface then these will show up on their screen as possible responses, which could be misleading

Reply:
Really it's fine